src/routes/JobsRoute.js

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { get } from 'lodash';
 import { getSASParams } from '@folio/stripes-erm-components';
-import { FormattedMessage } from 'react-intl';
 import { StripesConnectedSource } from '@folio/stripes/smart-components';
 import { stripesConnect } from '@folio/stripes/core';
 import { Callout } from '@folio/stripes/components';
+import calloutLogic from './components/calloutLogic';
```

Comment:
May not need this `calloutLogic` component if we are handling the callouts in `jobsRoute`

Reply:
See below